### ISSUE TITLE
Profiling information is not captured in tensorflow 2.0 version.

### ISSUE THREAD

**Issue by Priyankajaiswalintel** [tensorflow/tensorflow#37650]
I wanted to collect profiling information using current master tensorflow branch though the code for profiling is present inside Keras model. Still when I try to visualize the model in tensorboard its not collecting the profiling information.

Before the start of the test the message is thrown that the Profiler session started but when we load the logs to the tensorboard it shows empty profiling information.
profiler_session.cc:225] Profiler session started.

**Comment by ravikyram** [comment]
Please provide details about what platform you are using (operating system, architecture). Also include your TensorFlow version.

Request you to provide colab link or simple standalone code to reproduce the issue in our environment.It helps us in localizing the issue faster. 

Please, fill [Github issue Template](https://github.com/tensorflow/tensorflow/issues/new/choose).Thanks!

**Comment by Priyankajaiswalintel** [comment]
I am using on Ubuntu. My TF version is 2.0

**Comment by ravikyram** [comment]
@Priyankajaiswalintel 

Will it be possible to provide colab link or simple standalone code to reproduce the issue in our environment.It helps us in localizing the issue faster.Thanks!

**Comment by Priyankajaiswalintel** [comment]
The standalone code example:
import tensorflow.keras as keras
import tensorflow as tf
mnist = tf.keras.datasets.mnist
(x_train, y_train),(x_test, y_test) = mnist.load_data()
import matplotlib.pyplot as plt

plt.imshow(x_train[0],cmap=plt.cm.binary)
plt.show()
x_train = tf.keras.utils.normalize(x_train, axis=1)
x_test = tf.keras.utils.normalize(x_test, axis=1)
print(x_train[0])
plt.imshow(x_train[0],cmap=plt.cm.binary)
plt.show()
model = tf.keras.models.Sequential()
model.add(tf.keras.layers.Flatten())
model.add(tf.keras.layers.Dense(128, activation=tf.nn.relu))
model.add(tf.keras.layers.Dense(128, activation=tf.nn.relu))
model.add(tf.keras.layers.Dense(10, activation=tf.nn.softmax))
model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])
log_dir="/root/priyanka/profile"
tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=log_dir, profile_batch=3)
callbacks_list=[tensorboard_callback]
print("entering training")
model.fit(x_train, y_train, epochs=3,verbose=1,callbacks=callbacks_list)

I am trying to run the above code it gives me event file but where I am trying to load the generated logs to tensoboard its not giving me any profiling information when I press the profiling tab in tensorboard.
 tensorboard --logdir=/root/priyanka/profile/train/ --port=6009

**Comment by qiuminxu** [comment]
Looks like you already have a profile generated! 
The reason that you cannot see a trace maybe related to https://github.com/tensorflow/tensorboard/issues/3209
Can you relaunch your Chrome with --enable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports

Or you can try install the latest tb-nightly and tensorboard_plugin_profile and see if that fix your issue.
$ git clone https://github.com/tensorflow/profiler.git profiler
$ mkdir profile_env
$ python3 profiler/install_and_run.py --envdir=profile_env --logdir=/root/priyanka/profile/train/ --port=6009

**Comment by Priyankajaiswalintel** [comment]
when I try its giving me this error though
 python3 /root/priyanka/profiler/install_and_run.py --envdir=profile_env --logdir=/root/priyanka/profile/train/ --port=6007
usage: install_and_run.py [-h] --envdir ENVDIR --logdir LOGDIR
install_and_run.py: error: unrecognized arguments: --port=6007

**Comment by Priyankajaiswalintel** [comment]
I am able to get the profiling data by using the above method thank you.
